src/cim/components/Agent/AgentsSelectionHostCountAlerts.tsx

```diff
@@ -2,37 +2,44 @@ import React from 'react';
 import { Alert, AlertGroup, AlertVariant } from '@patternfly/react-core';
 import ShortCapacitySummary from '../ClusterDeployment/ShortCapacitySummary';
 import { AgentK8sResource } from '../../types';
+import { TFunction } from 'i18next';
 
-const getHostCountWarningText = (hostsSelected: number) => {
+const getHostCountWarningText = (hostsSelected: number, t: TFunction) => {
   switch (hostsSelected) {
     case 0:
-      return 'No host is selected.';
+      return t<string>('ui:No host is selected.');
```

Comment:
no need to specify `<string>` as that is the default 
```suggestion
      return t('ui:No host is selected.');
```

this applies for all other `t` usages